src/ServiceBus/ServiceBus.Autorest/11

```diff
@@ -0,0 +1 @@
+10
```

Comment:
And are these two files generated? 11 and 13

Reply:
Removed

Reply:
yes @Nickcandy , these are generated everytime. Not sure why.